src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -208,5 +210,73 @@ public string Value
         ///   Returns <see cref="JsonValueKind.String"/>
         /// </summary>
         public override JsonValueKind ValueKind { get => JsonValueKind.String; }
+
+        /// <summary>
+        ///   Converts the text value of this instance, which should encode binary data as base-64 digits, to an equivalent 8-bit unsigned <see cref="byte"/> array.
+        ///   The return value indicates wether the conversion succeeded.
+        /// </summary>
+        /// <param name="value">
+        ///   When this method returns, contains the <see cref="byte"/> array equivalent of the text contained in this instance,
+        ///   if the conversion succeeded.
+        /// </param>
+        /// <returns>
+        ///   <see langword="true"/> if text was converted successfully; othwerwise returns <see langword="false"/>.
+        /// </returns>
+        internal bool TryGetBytesFromBase64(out byte[] value)
+        {
+            Debug.Assert(_value != null);
+
+            // Shortest length of a base-64 string is 4 characters.
+            if (_value.Length < 4)
+            {
+                value = default;
+                return false;
+            }
+
+#if BUILDING_INBOX_LIBRARY
+            // we decode string -> byte, so the resulting length will
+            // be /4 * 3 - padding. To be on the safe side, keep padding and slice later
+            int bufferSize = _value.Length / 4 * 3;
+
+            byte[] arrayToReturnToPool = null;
+            Span<byte> buffer = bufferSize <= 256
```

Comment:
nit: Use `JsonConstants.StackallocThreshold`

```suggestion
            Span<byte> buffer = bufferSize <= JsonConstants.StackallocThreshold
```